packages/flutter_test/lib/src/all_elements.dart

```diff
@@ -38,11 +38,14 @@ Iterable<Element> collectAllElementsFrom(
 ///  4   5 6   7
 ///
 /// Will iterate in order 1, 3, 7, 6, 2, 5, 4. This avoids unnecessary
-/// allocation or CPU time, and performance is important here because this
-/// method is on the critical path for flutter_driver and
-/// package:integration_test performance tests.
+/// allocation or CPU time by maintaining a single [ListQueue] to track
+/// iteration and avoiding any reverse iteratino to go in let to right order.
```

Comment:
left to right order

Reply:
haha my "f" key is sticky :(

Reply:
iteration

Reply:
its okay, my `o` is almost completely gone on this macbook...